docs/concepts/what-is-a-schedule.md

```diff
@@ -0,0 +1,192 @@
+---
+id: what-is-a-schedule
+title: What is a Schedule
+sidebar_label: Schedule
+description: A Schedule enables the scheduling of Workflow Executions.
+tags:
+  - explanation
+  - term
+---
+
+A Schedule contains instructions for starting a [Workflow Execution](/concepts/what-is-a-workflow-execution) at specific times, and is meant to be a more flexible and user friendly approach over [Temporal Cron Jobs](/concepts/what-is-a-temporal-cron-job).
+
+- [How to enable Schedules](#how-to-enable-schedules)
+- [How to operate Schedules using tctl](/tctl/schedule/)
+
+A Schedule has identity, and is independent from a Workflow Execution.
+This is different from Temporal Cron Jobs, which rely on a cron schedule as a property of the Workflow Execution.
+
+### Action
+
+The Action of a Schedule is where the Workflow Execution properties are established, such as Workflow Type, Task Queue, parameters, Timeouts, etc.
+
+Workflow Executions started by a Schedule have the following additional properties:
+
+- The Action's timestamp is appended to the Workflow Id
```

Comment:
just fyi: this is true at the moment, but there has been some discussion about changing it so that if the workflow are not overlapping, the timestamp would not be appended. (if they are overlapping, it would have to be, of course, because of workflow id uniqueness.) the plan is to get feedback from users and make a final decision